### PR TITLE
updated docs and website styles

### PR DESCRIPTION
**Pull request by overthemike**
## Related Bug Reports or Discussions
N/A
Fixes #
N/A
## Summary
Updated some docs and styles for the website to make the two more consistent

## Check List

- [ x ] `pnpm run prettier` for formatting code and docs

### CODE REVIEW

**Comment by overthemike** [comment]
@dai-shi - not able to see the vercel deployment error logs - can you paste the error in here when you get a chance?

**Comment by dai-shi** [comment]
<details>

```
[05:26:13.884] Running build in Washington, D.C., USA (East) – iad1
[05:26:13.980] Cloning github.com/pmndrs/valtio (Branch: website-updates, Commit: 5f636e2)
[05:26:14.277] Cloning completed: 296.928ms
[05:26:17.557] Restored build cache from previous deployment (Gt69q9Pk2Mdcj1F1jSmVBFUVnooz)
[05:26:17.627] Running "vercel build"
[05:26:18.043] Vercel CLI 39.2.0
[05:26:18.446] Detected `pnpm-lock.yaml` version 6 generated by pnpm@8.x
[05:26:18.467] Installing dependencies...
[05:26:19.258] Lockfile is up to date, resolution step is skipped
[05:26:19.450] Already up to date
[05:26:20.064] 
[05:26:20.168] Done in 1.6s
[05:26:20.183] Detected Next.js version: 12.3.4
[05:26:20.203] Running "pnpm run build"
[05:26:20.719] 
[05:26:20.719] > website@ build /vercel/path0/website
[05:26:20.720] > next build
[05:26:20.720] 
[05:26:21.333] info  - Linting and checking validity of types...
[05:26:24.846] info  - Creating an optimized production build...
[05:26:26.384] Browserslist: caniuse-lite is outdated. Please run:
[05:26:26.384]   npx browserslist@latest --update-db
[05:26:26.384]   Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
[05:26:33.866] Browserslist: caniuse-lite is outdated. Please run:
[05:26:33.866]   npx update-browserslist-db@latest
[05:26:33.866]   Why you should do it regularly: https://github.com/browserslist/update-db#readme
[05:26:33.917] 
[05:26:33.918] [1m[33mwarn[39m[22m - As of Tailwind CSS v2.2, `lightBlue` has been renamed to `sky`.
[05:26:33.918] [1m[33mwarn[39m[22m - Update your configuration file to silence this warning.
[05:26:33.918] 
[05:26:33.918] [1m[33mwarn[39m[22m - As of Tailwind CSS v3.0, `warmGray` has been renamed to `stone`.
[05:26:33.918] [1m[33mwarn[39m[22m - Update your configuration file to silence this warning.
[05:26:33.918] 
[05:26:33.918] [1m[33mwarn[39m[22m - As of Tailwind CSS v3.0, `trueGray` has been renamed to `neutral`.
[05:26:33.918] [1m[33mwarn[39m[22m - Update your configuration file to silence this warning.
[05:26:33.919] 
[05:26:33.919] [1m[33mwarn[39m[22m - As of Tailwind CSS v3.0, `coolGray` has been renamed to `gray`.
[05:26:33.919] [1m[33mwarn[39m[22m - Update your configuration file to silence this warning.
[05:26:33.919] 
[05:26:33.919] [1m[33mwarn[39m[22m - As of Tailwind CSS v3.0, `blueGray` has been renamed to `slate`.
[05:26:33.919] [1m[33mwarn[39m[22m - Update your configuration file to silence this warning.
[05:26:35.752] info  - Compiled successfully
[05:26:35.752] info  - Collecting page data...
[05:26:37.071] info  - Generating static pages (0/36)
[05:26:37.909] ✘ [ERROR] [plugin esbuild-xdm] Expected the closing tag `</blockquote>` either after the end of `listItem` (194:17) or another opening tag after the start of `listItem` (193:1)
[05:26:37.909] 
[05:26:37.910]     _mdx_bundler_entry_point-6e83d7e2-3df8-4133-8ac5-b450728afc70.mdx:194:3:
[05:26:37.910]       194 │    </blockquote>
[05:26:37.910]           ╵    ~~~~~~~~~~~~~
[05:26:37.910] 
[05:26:37.910] 
[05:26:37.911] Error occurred prerendering page "/docs/api/basic/useSnapshot". Read more: https://nextjs.org/docs/messages/prerender-error
[05:26:37.911] Error: Build failed with 1 error:
[05:26:37.911] _mdx_bundler_entry_point-6e83d7e2-3df8-4133-8ac5-b450728afc70.mdx:194:3: ERROR: [plugin: esbuild-xdm] Expected the closing tag `</blockquote>` either after the end of `listItem` (194:17) or another opening tag after the start of `listItem` (193:1)
[05:26:37.911]     at failureErrorWithLog (/vercel/path0/website/node_modules/.pnpm/esbuild@0.14.54/node_modules/esbuild/lib/main.js:1624:15)
[05:26:37.911]     at /vercel/path0/website/node_modules/.pnpm/esbuild@0.14.54/node_modules/esbuild/lib/main.js:1266:28
[05:26:37.912]     at runOnEndCallbacks (/vercel/path0/website/node_modules/.pnpm/esbuild@0.14.54/node_modules/esbuild/lib/main.js:1046:63)
[05:26:37.912]     at buildResponseToResult (/vercel/path0/website/node_modules/.pnpm/esbuild@0.14.54/node_modules/esbuild/lib/main.js:1264:7)
[05:26:37.912]     at /vercel/path0/website/node_modules/.pnpm/esbuild@0.14.54/node_modules/esbuild/lib/main.js:1377:14
[05:26:37.912]     at /vercel/path0/website/node_modules/.pnpm/esbuild@0.14.54/node_modules/esbuild/lib/main.js:678:9
[05:26:37.913]     at handleIncomingPacket (/vercel/path0/website/node_modules/.pnpm/esbuild@0.14.54/node_modules/esbuild/lib/main.js:775:9)
[05:26:37.913]     at Socket.readFromStdout (/vercel/path0/website/node_modules/.pnpm/esbuild@0.14.54/node_modules/esbuild/lib/main.js:644:7)
[05:26:37.913]     at Socket.emit (node:events:517:28)
[05:26:37.913]     at addChunk (node:internal/streams/readable:368:12)
[05:26:38.545] info  - Generating static pages (9/36)
[05:26:39.161] info  - Generating static pages (18/36)
[05:26:39.552] info  - Generating static pages (27/36)
[05:26:39.721] info  - Generating static pages (36/36)
[05:26:39.722] 
[05:26:39.722] > Build error occurred
[05:26:39.724] Error: Export encountered errors on following paths:
[05:26:39.724] 	/docs/[...slug]: /docs/api/basic/useSnapshot
[05:26:39.724]     at /vercel/path0/website/node_modules/.pnpm/next@12.3.4_@babel+core@7.24.5_react-dom@17.0.2_react@17.0.2/node_modules/next/dist/export/index.js:404:19
[05:26:39.724]     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
[05:26:39.724]     at async Span.traceAsyncFn (/vercel/path0/website/node_modules/.pnpm/next@12.3.4_@babel+core@7.24.5_react-dom@17.0.2_react@17.0.2/node_modules/next/dist/trace/trace.js:79:20)
[05:26:39.725]     at async /vercel/path0/website/node_modules/.pnpm/next@12.3.4_@babel+core@7.24.5_react-dom@17.0.2_react@17.0.2/node_modules/next/dist/build/index.js:1230:21
[05:26:39.725]     at async Span.traceAsyncFn (/vercel/path0/website/node_modules/.pnpm/next@12.3.4_@babel+core@7.24.5_react-dom@17.0.2_react@17.0.2/node_modules/next/dist/trace/trace.js:79:20)
[05:26:39.725]     at async /vercel/path0/website/node_modules/.pnpm/next@12.3.4_@babel+core@7.24.5_react-dom@17.0.2_react@17.0.2/node_modules/next/dist/build/index.js:1092:17
[05:26:39.725]     at async Span.traceAsyncFn (/vercel/path0/website/node_modules/.pnpm/next@12.3.4_@babel+core@7.24.5_react-dom@17.0.2_react@17.0.2/node_modules/next/dist/trace/trace.js:79:20)
[05:26:39.725]     at async Object.build [as default] (/vercel/path0/website/node_modules/.pnpm/next@12.3.4_@babel+core@7.24.5_react-dom@17.0.2_react@17.0.2/node_modules/next/dist/build/index.js:65:29)
[05:26:39.757]  ELIFECYCLE  Command failed with exit code 1.
[05:26:39.777] Error: Command "pnpm run build" exited with 1
[05:26:40.121] 
```

</details>

File: docs/api/utils/derive.mdx [hunk 1]
```diff
@@ -7,6 +7,13 @@ description: 'create a new proxy derived from others'
 
 ## `derive`
 
+<blockquote className="note">
+ℹ️ &nbsp; Note
+
+There is a new library called [valtio-reactive](https://github.com/valtiojs/valtio-reactive) that is a drop-in replacement for `derive` and provides a more powerful API. It is recommended to use that library instead.
```
Comment (dai-shi) [inline docs/api/utils/derive.mdx:13]: It's not a drop-in replacement. It's purpose and implementation design is a bit different.
`watch` will be deprecated in favor of `valtio-reactive`.

So, I'd suggest to remove this section for now.

**Review by dai-shi — CHANGES_REQUESTED** [review]
small requests

**Review by dai-shi — APPROVED** [review]

File: docs/resources/libraries.mdx [hunk 2]
```diff
@@ -10,12 +10,17 @@ Valtio provides bare necessities for proxy state management which is great for m
...
+<blockquote className="warning">
+⚠️ &nbsp; Warning
+
+Disclaimer: These libraries may have bugs, limited maintenance, or other limitations and are not officially recommended by pmndrs or the valtio maintainers. This list is to provide a good starting point for someone looking to extend valtio's feature set.
+
+</blockquote>
 
 <br />
 
+- [valtio-reactive](https://github.com/valtiojs/valtio-reactive) - Valtio-reactive makes valtio a reactive library.
```
Comment (dai-shi) [inline docs/resources/libraries.mdx:22]: oops, it's not ordered alphabetically.

**Pull request merged**